ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/button.html

```diff
@@ -0,0 +1,9 @@
+<div class='${properties.buttonType}${properties.disabledButton}'>
```

Comment:
@rakeshkumar-verma ; why do we have this div wrapping <button>

Reply:
added based previous approach. Now removed this as we are handling button type from style system